vocabularies/Hierarchy.xml

```diff
@@ -228,6 +245,91 @@ the following collections of hierarchy nodes are distinguished:
         <ReturnType Type="Collection(Edm.EntityType)" />
       </Function>
 
+      <Action Name="Template_CreateAction" IsBound="true" EntitySetPath="HierarchyNodes">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Template for actions that create a new node and are named in [`RecursiveHierarchy/CreateAction`](#RecursiveHierarchyType)" />
+        <Parameter Name="HierarchyNodes" Type="Collection(Edm.EntityType)" Nullable="false" />
+        <Parameter Name="HierarchyQualifier" Type="Aggregation.HierarchyQualifier" Nullable="false" />
+        <Parameter Name="Entity" Type="Edm.EntityType" Nullable="false">
+          <Annotation Term="Core.Description" String="The new node to be created" />
+          <Annotation Term="Core.LongDescription">
+            <String>If the entity exists already without a node identifier for the recursive hierarchy,
+            it is upserted with PATCH semantics.
+            It is an error if the entity exists already with a node identifier for the recursive hierarchy.</String>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Node" Type="Edm.PrimitiveType" Nullable="false">
+          <Annotation Term="Core.Description" String="The node identifier, defaults to the node property of the `Entity`" />
+          <Annotation Term="Core.OptionalParameter">
+            <Record/>
+          </Annotation>
+        </Parameter>
+        <Parameter Name="Parent" Type="Edm.PrimitiveType" Nullable="true">
+          <Annotation Term="Core.Description" String="Node identifier of the new node's parent (null if the new node shall be a root)" />
+        </Parameter>
+        <Parameter Name="SiblingRank" Type="Edm.Int64" Nullable="true">
+          <Annotation Term="Core.Description" String="Optional integer rank of the new node among its siblings" />
+        </Parameter>
```

Comment:
Or rather the node identifier of the `RightSibling` (null means new node becomes the rightmost sibling)?

Reply:
Would specifying the node before or after it in the list, mean that the client does not need to know about the SiblingRank? If so, I am in favor of this simplification.